front/src/modules/conflict/components/ConflictsList.tsx

```diff
@@ -6,9 +6,9 @@ import { sec2time } from 'utils/timeManipulation';
 
 export type Conflict = {
```

Comment:
Can you remove this type and use the new one from osrdEditoastApi ? :) Also you should use the new type in Timetable.tsx too